src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -36,8 +36,11 @@ def get_device_id_completion_list(prefix, action, parsed_args, **kwargs):#pylint
 # Arguments for 'iot hub policy' group
 register_cli_argument('iot hub policy', 'policy_name', options_list=('--name', '-n'), help='Share access policy name.')
 
+# Arguments for 'iot hub job' group
+register_cli_argument('iot hub job', 'job_id', help='Job Id.')
+
 # Arguments for 'iot hub create'
-register_cli_argument('iot hub create', 'hub_name', options_list=('--name', '-n'), completer=None)
+register_cli_argument('iot hub create', 'hub_name', completer=None)
```

Comment:
A general comment for all of IoT is that you don't support the `--ids` parameter. If you append the correct `id_part` kwarg to the aliases in your _params.py file, the infrastructure will handle this for you. I wouldn't hold up this PR for it--should probably open a separate issue and address it all at once.